Library/OcCpuLib/OcCpuLib.c

```diff
@@ -424,16 +487,31 @@ ScanIntelProcessor (
       TimerAddr = InternalGetPmTimerAddr (&TimerSourceType);
       DEBUG ((DEBUG_INFO, "OCCPU: Timer address is %Lx from %a\n", (UINT64) TimerAddr, TimerSourceType));
       DEBUG_CODE_END ();
-      Cpu->CPUFrequencyFromTSC = InternalCalculateTSCFromPMTimer (Recalculate);
+      Cpu->CPUFrequencyFromApple = InternalCalculateTSCFromApplePlatformInfo (NULL);
+      Cpu->CPUFrequencyFromTSC   = InternalCalculateTSCFromPMTimer (Recalculate);
```

Comment:
THis is broken in another way now. We do not want to call `InternalCalculateTSCFromPMTimer` if Apple succeeds.